packages/twenty-server/src/metadata/field-metadata/dtos/relation-definition.dto.ts

```diff
@@ -0,0 +1,41 @@
+import { Field, ObjectType, registerEnumType } from '@nestjs/graphql';
+
+import { IsEnum, IsNotEmpty } from 'class-validator';
+
+import { FieldMetadataDTO } from 'src/metadata/field-metadata/dtos/field-metadata.dto';
+import { ObjectMetadataDTO } from 'src/metadata/object-metadata/dtos/object-metadata.dto';
+import { RelationMetadataDTO } from 'src/metadata/relation-metadata/dtos/relation-metadata.dto';
+
+export enum RelationDefinitionType {
+  'MANY_TO_ONE',
+  'ONE_TO_MANY',
+  'ONE_TO_ONE',
+}
```

Comment:
What about this instead ?
```suggestion
export enum RelationDefinitionType {
  ONE_TO_ONE = RelationMetadataType.ONE_TO_ONE,
  ONE_TO_MANY = RelationMetadataType.ONE_TO_MANY,
  MANY_TO_MANY = RelationMetadataType.MANY_TO_MANY,
  MANY_TO_ONE = 'MANY_TO_ONE',
}
```
Just to keep track that we're extending `RelationMetadataType` enum, and maybe later we'll merge them into only one enum.

Reply:
Nice !